ui/src/styles/_cta.scss

```diff
@@ -59,6 +59,9 @@
     --buttonPadding: 5px 25px;
     font-size: var(--smallText);
   }
+  &--footer {
+    width: 11em;
+  }
```

Comment:
im thinking we need to double check the global use case of buttons. maybe we just want to give all buttons a min width for example? anyways, lets just leave the buttons as-is for now and remove this. we can circle back when we understand the global cta styles a bit better